code/script.js

```diff
@@ -222,84 +223,139 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log("secret ", secret);
 }
 
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard();
+  setSecret();
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
 
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
+  const category = questions.options[questions.selectedIndex].parentNode.label; //option group(category)
+  const value = questions.options[questions.selectedIndex].value; //value of the question we've selected.
 
+  console.log("category ", category)
+  console.log("value ", value)
   currentQuestion = {
-    category: category,
-    // value: value
+    category: category, //hair-eyes-accesories-other
+    value: value //blue-green-yellow
```

Comment:
Very nice that you have left comments to understand and remember  what is what in the code.